### PR TITLE
ISSUE-707: Upgrade to Dropwizard 1.3.22

### PR DESCRIPTION
**Pull request by viktorsomogyi**
Addresses issue #707

### CODE REVIEW

**Comment by viktorsomogyi** [comment]
retest this please

**Comment by akatona84** [comment]
typo in title, version is 1.3.14

**Comment by viktorsomogyi** [comment]
As per conversations with infosec, we should upgrade to the latest 1.3 DW, so changing this PR to upgrade to 1.3.22.

File: webservice/pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <artifactId>registry-webservice</artifactId>
 
     <properties>
-        <dropwizard.metrics.version>3.1.2</dropwizard.metrics.version>
+        <dropwizard.metrics.version>4.0.5</dropwizard.metrics.version>
         <dropwizard.swagger.version>1.2.4-1</dropwizard.swagger.version>
```
Comment (akatona84) [inline webservice/pom.xml:16]: as per the compatibility matrix of this dw swagger (https://github.com/smoketurner/dropwizard-swagger#version-matrix)
it should be updated too to 1.3.17-1  is the latest 1.3.x

Comment (viktorsomogyi) [reply webservice/pom.xml:16]: Thanks, I didn't notice this compatibility matrix.

**Pull request merged**